src/test/java/org/opensearch/security/test/helper/cluster/ClusterConfiguration.java

```diff
@@ -79,11 +78,7 @@ public enum ClusterConfiguration {
     CLIENTNODE(new NodeSettings(true, false), new NodeSettings(false, true), new NodeSettings(false, true), new NodeSettings(false, false)),
 
     // 3 nodes (1m, 2d) plus additional UserInjectorPlugin
-    USERINJECTOR(
-        new NodeSettings(true, false, Lists.newArrayList(UserInjectorPlugin.class)),
-        new NodeSettings(false, true, Lists.newArrayList(UserInjectorPlugin.class)),
-        new NodeSettings(false, true, Lists.newArrayList(UserInjectorPlugin.class))
-    );
+    USERINJECTOR(new NodeSettings(true, false), new NodeSettings(false, true), new NodeSettings(false, true));
```

Comment:
for my understanding, what was the purpose of UserInjector class in NodeSettings?

Reply:
USERINJECTOR is a cluster configuration for testing. It means that `plugins.security.unsupported.inject_user.enabled` is set to `true` which is an unsecure setting that let's a user pass a serialized user object as an http header and inject it into the threadcontext.

Reply:
Nit: I think we need to deprecate `lugins.security.unsupported.inject_user.enabled` and remove it.

Reply:
This is a good follow up issue